slideextract.cpp

```diff
@@ -159,19 +159,22 @@ se_extract_slides(const char *file, const char *outprefix, struct roi *roi)
 
 	do {
 		capture.read(current);
-		if (current.empty())
-			break;
 
-		if (!last.empty()) {
-			double cmp = _se_compare_image (last, current, roi);
-			if (cmp <= 0.999) {
-				snprintf (str, sizeof(str)/sizeof(*str), "%s%d.png", outprefix, num++);
-				imwrite(str, last, image_properties);
-			}
+		bool is_final_frame = current.empty();
+		bool is_changed_frame = !current.empty() && !last.empty()
+			&& _se_compare_image (last, current, roi) <= 0.999;
+                
```

Comment:
White space

Reply:
What do you mean by this comment? To remove an empty line?

Reply:
The empty line is indented (not visible on GitHub) but it lights up in red when looking at the diff on the command line. Nothing serious.